stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2923,11 +2993,15 @@ internal struct _Native${Self}Storage<${TypeParameters}> {
     return res
   }
 
+  /// Gets the key at the given Index, and then bridges it.
```

Comment:
This should be “Returns the key at the given Index, bridged.“  Again, see the guidelines.